react-example/src/components/EventsForm.tsx

```diff
@@ -52,9 +33,12 @@ export const EventsForm: FC<Props> = ({ method, endpointName, heading }) => {
   const handleTrack = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     setTrackingEvent(true);
+
+    const conditionalParams = needsEventName
+      ? { eventName: trackEvent }
+      : { messageId: trackEvent };
     method({
-      eventName: trackEvent,
-      messageId: 'mock-id',
```

Comment:
this was changed a little bit as well because the `/track` endpoint needs an event name but the other ones such as `/trackInAppClose` only need a messageID and not an event name